### PR TITLE
JBOSS config section added

### PR DESCRIPTION
**Pull request by YogiSoni-lazy**

### CODE REVIEW

File: modules/config/pages/eap-cli.adoc [hunk 1]
```diff
@@ -1,5 +1,189 @@
 = JBoss EAP CLI
```
Comment (rsriniva) [inline modules/config/pages/eap-cli.adoc:1]: I am assuming this file is just your high level plan and the content is not part of the PR. SO I am going to  ignore this...

File: modules/config/pages/eap-config.adoc [hunk 2]
```diff
@@ -1,11 +1,159 @@
 = JBoss EAP Configuration
 
-Brief blurb about the different ways to configure JBoss EAP....
+== Overview of Standalone Server
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: It's Standalone mode. Call it JBoss EAP Standalone mode

File: modules/config/pages/eap-config.adoc [hunk 3]
```diff
@@ -1,11 +1,159 @@
...
 
-== Section Header
+* Standalone servers are perfect for running a single instance of Red Hat JBoss EAP.
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc:5]: Add a bullet - you can run multiple standalone instances on the same server or across different servers. Each server is managed "standalone", that is, individually.

File: modules/config/pages/eap-config.adoc [hunk 4]
```diff
@@ -1,11 +1,159 @@
...
+* All configurations are managed in the `JBOSS_HOME/standalone/configuration/standalone.xml` file.
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: not true. standalone can also run standalone-ha, standalone-full and full-ha.xml

File: modules/config/pages/eap-config.adoc [hunk 5]
```diff
@@ -1,11 +1,159 @@
...
+* For instance, when starting the server with standalone.sh:
++
+[subs="+quotes,+macros"]
+----
+$ ./standalone.sh -Djboss.server.base.dir=/path/to/base/directory -Djboss.home.dir=/path/to/home/directory
+----
+
+== Lab: Run a standalone JBoss EAP 8 server using a custom location for the server base directory.
+
+*Outcome*: In this lab, you should be able to run a standalone JBoss EAP 8 server using a custom location for the server base directory.
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: Change to "You should be able to run two instances of JBoss EAP on the same server using....

File: modules/config/pages/eap-config.adoc [hunk 6]
```diff
@@ -1,11 +1,159 @@
...
+----
+[vagrant@server ~]$ cd /opt/EAP-8.0.0/standalone/
+[vagrant@server standalone]$ sudo cp -r configuration deployments lib /opt/standalone-running
+----
+
+* Modify the standalone server port offset.
+
+. To verify that the JBoss EAP server is utilizing the new, copied configuration files, update the port offset. You can run multiple instances of EAP, by using the port offset to avoid port conflicts.
+
+. Update the new configuration file by editing the standalone.xml file to verify that JBoss EAP is using the configuration files in the new directory. Open the `/opt/standalone-running/configuration/standalone.xml` file with a text editor.
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: You are violating the NOTE you mentioned earlier in the section - do not edit XML directly!

File: modules/config/pages/eap-config.adoc [hunk 7]
```diff
@@ -1,11 +1,159 @@
...
+
+. In the standard-sockets socket binding group, change the default value of the port-offset attribute from 0 to 10000. This adds 10,000 to every port number used in the standard-sockets binding group.
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: No need to do this. You can pass a -D flag to standalone.sh

File: modules/config/pages/eap-config.adoc [hunk 8]
```diff
@@ -1,11 +1,159 @@
...
+...
+<socket-binding-group name="standard-sockets" default-interface="public" port-offset="${jboss.socket.binding.port-offset:10000}">
+...
+----
+
+Save your changes to the standalone.xml file, and close the text editor.
+
+* Run and test the JBoss EAP Server.
+
+. Run the following command to start the JBoss EAP server by using the standalone.sh script in the original JBoss EAP installation, but using the new configuration files:
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: Pass another -D flag, -Djboss.socket.binding.port-offset:10000

Comment (rsriniva) [reply modules/config/pages/eap-config.adoc]: check AD248 for the correct flag

File: modules/config/pages/eap-config.adoc [hunk 9]
```diff
@@ -1,11 +1,159 @@
...
+----
+[vagrant@server ~]$ ls /opt/standalone-running/
+configuration  data   deployments  lib  log  tmp
+----
+
+Notice the three data, log, and tmp new directories . These directories are automatically created when the JBoss EAP server starts.
+
+* Clean Up
+
+Stop the running instance of JBoss EAP that was started in the previous step. Press `Ctrl+C` in the terminal window in which the server is running.
```
Comment (rsriniva) [inline modules/config/pages/eap-config.adoc]: I wonder if this is right section to talk about the port binding flag - that is, how to bind and run EAP on a NIC which has a public IP instead of localhost. Maybe one more lab in this section - a short one showing how to bind different IPs for HTTP traffic and management. See ./standalone.sh --help and product docs for the various other flags. See what else is important and can be covered in this section

**Review by rsriniva — CHANGES_REQUESTED** [review]
Excellent start! PLease address PR comments before approval.

File: modules/config/pages/eap-cli.adoc (hunk 1 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-cli.adoc:1]: yes ignore this file

File: modules/config/pages/eap-config.adoc (hunk 2 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-config.adoc]: updated the title

File: modules/config/pages/eap-config.adoc (hunk 7 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-config.adoc]: updated the steps

File: modules/config/pages/eap-config.adoc (hunk 6 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-config.adoc]: removed this section

File: modules/config/pages/eap-config.adoc (hunk 8 above)
Comment (YogiSoni-lazy) [reply modules/config/pages/eap-config.adoc]: updated the changes

**Review by rsriniva — APPROVED** [review]
Let's merge this and move on. will revisit later for minor edits
